### PR TITLE
#247-ShreeSinghal-TestgetAllChangeRequests

### LINKED ISSUE #247: Backend - Test getAllChangeRequests Endpoint
### Description

Write unit tests for the getAllChangeRequests endpoint. To work on this ticket, you will need to read through the endpoint to learn how it works.

### Acceptance Criteria

test that it works

### Proposed Solution

Check out this epic https://github.com/Northeastern-Electric-Racing/FinishLine/issues/238 for resources and tips on how to write these tests.

### Mocks

_No response_

### PR DESCRIPTION
**Pull request by shreesinghal**
## Changes

created a test for getAllChangeRequests

## Test Cases

it works when there are change requests

## To Do

possibly more tests needed

## Checklist

It can be helpful to check the `Checks` and `Files changed` tabs.
Please review the [contributor guide](https://nerdocs.atlassian.net/wiki/spaces/NER/pages/8060929/Software+Contributor+Guide) and reach out to your Tech Lead if anything is unclear.
Please request reviewers and ping on slack only after you've gone through this whole checklist.

- [x] All commits are tagged with the ticket number
- [x] No linting errors / newline at end of file warnings
- [x] All code follows repository-configured prettier formatting
- [x] No merge conflicts
- [x] All checks passing
- [x] Screenshots of UI changes (see Screenshots section)
- [x] Remove any non-applicable sections of this template
- [x] Assign the PR to yourself
- [x] No `yarn.lock` changes (unless dependencies have changed)
- [x] Request reviewers & ping on Slack
- [x] PR is linked to the ticket (fill in the closes line below)

Closes #247

### CODE REVIEW

File: src/frontend/src/pages/ProjectsPage/HCPC.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+public class Main {
```
Comment (ryanhowe28) [inline src/frontend/src/pages/ProjectsPage/HCPC.java:1]: Did you create this file/class?

Comment (shreesinghal) [reply src/frontend/src/pages/ProjectsPage/HCPC.java:1]: Nope, I didn't create it - only lines 56 to 66

Comment (ryanhowe28) [reply src/frontend/src/pages/ProjectsPage/HCPC.java:1]: Can you remove it ?

Comment (shreesinghal) [reply src/frontend/src/pages/ProjectsPage/HCPC.java]: I just removed it. Seems I had accidentally added the file before.

Comment (ryanhowe28) [reply src/frontend/src/pages/ProjectsPage/HCPC.java]: Thanks!

**Review by ryanhowe28 — APPROVED** [review]
Looks good!

**Review by anthonybernardi — APPROVED** [review]
awesome!!!!

**Pull request merged**